src/resources/Catalogs/CatalogInterfaces.ts

```diff
@@ -14,6 +14,12 @@ export type HierarchyWithFields<T extends {fields?: string[]}> = Omit<T, 'fields
     fields: string[];
 };
 
+export interface Cached<T> {
```

Comment:
Not sure about that, I know this is to reflect the back-end, but `Cached` would still be exported globally from this project :/

I think it would be best to expose a `CachedCatalogFieldsModel` here instead, that uses this exact interface, but without the `export`.

So:

```
interface Cached<T> {
    item: T;
    lastUpdated: Date;
    nextUpdate: Date;
}

export type CachedCatalogFieldsModel = Cached<CatalogFieldsModel>;
```

Reply:
smart ☝️

Reply:
Great idea! Should we also not export `CatalogFieldsModel` in that case?